.github/AI_USAGE_POLICY.md

```
@@ -0,0 +1,144 @@
+    * Minimum of one human approval
+    * Signed commits
+    * Passing CI checks
+    * Force-pushing to protected branches is prohibited
+
+### 4.3 CWE-655 Mitigation
+
+ * 4.3.A dual-approval system
+    * The project implements a dual-approval system to help prevent single points of failure.
+    * AI approvals are tracked separately from human approvals in the review process. Humans
+must be responsible for the actual merge of pull-requests.
+    * Every user (eg. AI, or human) must have a distinct code-signing identity (see 4.1.B).
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)